src/main/rx/sumd.c

```diff
@@ -156,7 +160,7 @@ bool sumdInit(const rxConfig_t *rxConfig, rxRuntimeConfig_t *rxRuntimeConfig)
 {
     UNUSED(rxConfig);
 
-    rxRuntimeConfig->channelCount = SUMD_MAX_CHANNEL;
+    rxRuntimeConfig->channelCount = MAX_SUPPORTED_RC_CHANNEL_COUNT;
```

Comment:
Maybe this should be `MIN(SUMD_MAX_CHANNEL , MAX_SUPPORTED_RC_CHANNEL_COUNT);` ?

Reply:
Ah I see, if one day Betaflight supports more than 32 channels.
Yes good point.